package/MDAnalysis/topology/TOPParser.py

```diff
@@ -277,27 +285,18 @@ def next_getter():
         attrs['dihedrals'], attrs['impropers'] = self.parse_dihedrals(
                               attrs.pop('diha'), attrs.pop('dihh'))
 
-        # Guess elements if not in topology or if any dummy atoms are found
-        if not 'elements' in attrs:
-            msg = ("ATOMIC_NUMBER record not found, guessing atom elements "
-                   "based on their atom types")
+        # Warn user if elements not in topology
+        if 'elements' not in attrs:
+            msg = ("ATOMIC_NUMBER record not found, elements attribute will "
+                   "not be populated")
```

Comment:
Could you add something about `guess_elements` (or whatever the method's called?)

Reply:
Warnings now direct users to MDAnalysis.topology.guessers in 95cb49e